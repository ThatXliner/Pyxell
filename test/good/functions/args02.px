
func mul(Int a, Int b: 1, Int c: 1, Bool neg: false) Int def
    return a * b * c * (neg ? -1 : 1)

print mul(5), mul(3, -4)
print mul(-10, -100, -1000, true)
