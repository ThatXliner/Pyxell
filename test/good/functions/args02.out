5 -12
1000000
